### PR TITLE
Add fastify-firebase-auth to the fastify community

### PR DESCRIPTION
**Pull request by oxsav**
Added a Url to a new fastify plugin for Firebase Authentication.
Pls give some feedback about it

### CODE REVIEW

File: docs/Ecosystem.md [hunk 1]
```diff
@@ -37,3 +37,4 @@ Plugins maintainted by the fastify team are listed under [Core](#core) while plu
 - [`fastify-session`](https://github.com/SerayaEryn/fastify-session) a session plugin for Fastify.
 - [`fastify-sse`](https://github.com/lolo32/fastify-sse) to provide Server-Sent Events with `reply.sse( … )` to Fastify.
 - [`fastify-ws`](https://github.com/gj/fastify-ws) WebSocket integration for Fastify — with support for WebSocket lifecycle hooks instead of a single handler function. Built upon [ws](https://github.com/websockets/ws) and [uws](https://github.com/uNetworking/bindings/tree/master/nodejs).
+- [`fastify-firebase-auth`](https://github.com/oxsav/fastify-firebase-auth) Firebase Authentication for Fastify - Support all the methods relate to the authentication API.
```
Comment (jsumners) [inline docs/Ecosystem.md]: Please alphabetize the list.

**Comment by oxsav** [comment]
Done 👍

File: docs/Ecosystem.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ Plugins maintainted by the fastify team are listed under [Core](#core) while plu
 
 #### [Community](#community)
 - [`fastify-apollo`](https://github.com/coopnd/fastify-apollo) Run an [Apollo Server](https://github.com/apollographql/apollo-server) to serve GraphQL with Fastify.
+- [`fastify-firebase-auth`](https://github.com/oxsav/fastify-firebase-auth) Firebase Authentication for Fastify - Support all the methods relate to the authentication API.
```
Comment (jsumners) [inline docs/Ecosystem.md]: How about "Firebase authentication for Fastify supporting all of the methods relating to the authentication API."?

Comment (oxsav) [reply docs/Ecosystem.md]: Seems better :) . Modified it already. Don't know if really want to merge it regarding this thread: https://github.com/oxsav/fastify-firebase-auth/issues/1 

Let me know what do you think

Comment (delvedor) [reply docs/Ecosystem.md]: I think we can work on this plugin and once is ready we can merge this pr.
What do you think? :)

Comment (oxsav) [reply docs/Ecosystem.md]: Sorry for late response. Yep I agree! thanks for the feedback ;)

**Pull request merged**